examples/0_Simple/simpleStreams/Makefile

```diff
@@ -1,3 +1,3 @@
 build:
-	gprbuild -p -Pdevice
-	gprbuild -p -Phost -largs $(PWD)/lib/*.fatbin.o
+	gprbuild -P device.gpr
+	gprbuild -P host -j0 -largs $(PWD)/lib/*.fatbin.o
```

Comment:
why add the .gpr extension only on the first line?

Reply:
(and looking at this, -j0 is confusing and unnecessary for examples)